mackup/applications/vivaldi.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Vivaldi
+
+[configuration_files]
+Library/Application Support/Vivaldi
```

Comment:
Has it been tested? Isn't there a lot of cache in here?

Reply:
> Has it been tested? Isn't there a lot of cache in here?

Actually, caches are located in Library/Caches/Vivaldi